app-sso/about.hbs.md

```diff
@@ -19,3 +18,37 @@ configured secure storage, and client restrictions.
 
 AppSSO's authorization server is based off of Spring Authorization Server project.
 For more information, see [Spring documentation](https://spring.io/projects/spring-authorization-server).
+
+## How this documentation is organized
```

Comment:
**[praise]** that's a great intro

Reply:
Credit goes to [STK's landing page](https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Application-Platform/1.6/tap/services-toolkit-about.html)